src/models/paymentRequest.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export async function createPaymentRequest(requestBody) {
+  try {
+      if (requestBody.CardNumber === '' || requestBody.CardNumber === undefined) {
+        console.warn("got empty value for card number")
+        throw new Error('Please enter card number')
+      }
```

Comment:
We don't really need this, because we can just specify that the form cannot be submitted with empty fields. However, if you wish to include it anywany, you should probably do it for every other field in that form...